jwst/associations/tests/data/rules_level3_coron.py

```diff
@@ -17,11 +17,15 @@
 # --------------------------------
 # Start of the User-level rules
 # --------------------------------
-
+'''
```

Comment:
*facepalm*. this will be fixed in #2750

Reply:
Actually something happened here. The file did not start out with this double definition. The `class` line of the first part is correct with the inheritance. The second `class` line should not be there.

Reply:
Ah, I see. I removed the inheritance because
```
data/rules_level3_coron.py:23: in <module>
    AsnMixin_OpticalPath,
E   NameError: name 'AsnMixin_OpticalPath' is not defined
data/rules_level3_coron.py:23: in <module>
    AsnMixin_Base
E   NameError: name 'AsnMixin_Base' is not defined
```
And I don't see them defined anywhere. Any suggestions?

Reply:
After full investigation, this file, `rules_level3_coron.py`, is not used anymore. At this point it is best to remove it permanently. Would explain all the wonky.

Reply:
file removed!